tests/test_sofastream.py

```diff
@@ -0,0 +1,68 @@
+from sofar import SofaStream
+import sofar as sf
+from tempfile import TemporaryDirectory
+from pytest import raises
+import pytest
+import netCDF4
+import numpy as np
+import os
+
+# Temporary SOFA-file
+@pytest.fixture
+def temp_sofa_file(tmp_path_factory):
+
+    filename = tmp_path_factory.mktemp("data") / "test_sofastream.sofa"
+    sofa = sf.Sofa("SimpleFreeFieldHRIR")
+    sofa.Data_IR = np.array([[0, 1], [2, 3], [4, 5]])
+    sofa.GLOBAL_RoomType = "free field"
+    sofa.Data_SamplingRate_Units = "hertz"
+    sf.write_sofa(filename, sofa)
+    return filename
```

Comment:
The fixture should go into conftest.py in the same directory. You can check it out in the pyfar package